ui/projects/DeployProject.qml

```diff
@@ -0,0 +1,54 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+
+import "../../backend/core"
+import "../../backend/sccs" as Backend
+
+Item {
+    id: root
+
+    property var projectIndex
+
+    width: scroll.width
+    height: 110 + (130 * Store.projects_project_settings.projectObj.projects[projectIndex].repositories.length)
```

Comment:
will trigger an error if projetIndex is undefined as it is the case

Reply:
The component will not be displayed if index undefined, a parent component take care of this issue.